test/intern/mixins/ColumnSet.js

```diff
@@ -94,4 +128,77 @@ define([
 			grid.focus(grid.cell(1, '1-0-1'));
 		});
 	});
+
+	test.suite('ColumnSet + Tree mixins', function () {
+		test.before(function () {
+			var TreeColumnSetGrid = declare([ OnDemandGrid, ColumnSet, Tree ]);
+			var data = [];
+			var store;
+
+			for (i = 0; i < 5; i++) {
+				var parentId = '' + i;
+				data.push({
+					id: parentId,
+					value: 'Root ' + i,
+					name: 'Root ' + i,
+					type: 'country'
+				});
+				for (k = 0; k < 100; k++) {
+					data.push({
+						id: i + ':' + k,
+						parent: parentId,
+						value: 'Child ' + k,
+						name: 'Child ' + i,
+						type: 'city',
+						hasChildren: false
+					});
+				}
+			}
+
+			store = createHierarchicalStore({
+				data: data
+			});
+
+			grid = new TreeColumnSetGrid({
+				collection: store,
+				enableTreeTransitions: false,
+				columnSets: [[
+					[ {renderExpando: true, label:"Name", field:"name", sortable: false} ],
+					[ {label:"Type", field:"type", sortable: false} ]
+				]]
+			});
+
+			document.body.appendChild(grid.domNode);
+			grid.startup();
+		});
+
+		test.afterEach(function () {
+			for (var i = handles.length; i--;) {
```

Comment:
No handles are registered in this test suite at all.  Also, I'd probably just change the before and after functions to beforeEach and afterEach for simplicity here - it's irrelevant for one test, but you'd have to reset the expanded state between tests otherwise if we added more.